lexibank_chacontukanoan.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+import pylexibank
+
+from clldutils.misc import slug
+
+from pylexibank.util import getEvoBibAsBibtex
+
+
+# Customize your basic data.
+# if you need to store other data in columns than the lexibank defaults, then over-ride
+# the table type (pylexibank.[Language|Lexeme|Concept|Cognate|]) and add the required columns e.g.
+#
+# import attr
+#
+# @attr.s
+# class Concept(pylexibank.Concept):
+#    MyAttribute1 = attr.ib(default=None)
+
+
+class Dataset(pylexibank.Dataset):
+    dir = Path(__file__).parent
+    id = "chacontukanoan"
+
+    # register custom data types here (or language_class, lexeme_class, cognate_class):
+    # concept_class = Concept
+
+    # define the way in which forms should be handled
+    form_spec = pylexibank.FormSpec(
+        brackets={"(": ")"},  # characters that function as brackets
+        separators=";/,",  # characters that split forms e.g. "a, b".
+        missing_data=('?', '-'),  # characters that denote missing data.
+        strip_inside_brackets=True
```

Comment:
to be on the save side, please add "first_form_only=True"

Reply:
wait: if we add the form, it means, we do not have a formspec, so the form-spec won't do anything, so you can just leave it blank. The whole "form_spec" line.

Reply:
ok